### DISCUSSION TITLE
Hello, everyone, I also worked on a Razor Pages CleanArchitecture Project

### DISCUSSION THREAD

**Discussion by neozhu** [jasontaylordev/CleanArchitecture#636] [General]
https://github.com/neozhu/RazorPageCleanArchitecture.git
Reference this project and blazorhero/CleanArchitecture project.
The front-end is replaced with a Razor Page(simplest), please review it.

**Comment by litan1106** [comment]
Nice. do you have the file changes in a PR? (so we can track what files changed)

**Comment by HybridSolutions** [comment]
You did it as this project could have been upgraded. Not using startup.cs anymore is a very nice addition!
